packages/@okta/vuepress-site/docs/reference/csi-delauth-hook/index.md

```diff
@@ -0,0 +1,536 @@
+---
+title: Custom Source Integration Delegated Authentication Inline Hook Reference
+excerpt: Add custom logic to the Custom Source Integration Delegated Authentication process.
+---
+
+# Custom Source Integration Delegated Authentication Inline Hook Reference
+
+<ApiLifecycle access="beta" />
+
+This page provides reference documentation for:
+
+- JSON objects contained in the outbound request from Okta to your external service
+
+- JSON objects you can include in your response
+
+This information is specific to the Custom Source Integration Delegated Authentication Inline Hook, one type of Inline Hook supported by Okta.
+
+## See also
+
+For a general introduction to Okta Inline Hooks, see [Inline Hooks](/docs/concepts/inline-hooks/).
+
+For information on the API for registering external service endpoints with Okta, see [Inline Hooks Management API](/docs/reference/api/inline-hooks/).
+
+## About
+
+The Custom Source Integration Delegated Authentication Inline Hook enables you to delegate authentication of a user to an external identity source.
+
+The hook supports a few operations. It first allows you to import a user into Okta after a successful authentication. From there it allows delegated authentication of the user to the external source, and fetches the latest profile of the user from the external source.
+
+Each of these operations corresponds to an individual inline hook invocation, and each corresponding request is tagged with a `requestType` property in the request headers.
+
+The following sections describe each of these operations and the objects in their requests and responses.
+
+## User attributes
+
+To support user lifecycle and credential lifecycle operations, we expect the external source to provide certain attributes:
+
+- `status`: The status of the user, with a value of ACTIVE or DISABLED.
+- `passwordExpiryTime`: The time when the user's password expires, in the form of Epoch time.
+- `lastUpdate`: When the user's profile was last updated in the external source.
```

Comment:
do we need to specify a format for this value?

Reply:
Good point, using the Epoch time like passwordExpiryTime would be good. Let me update.